packages/core/src/utils.ts

```diff
@@ -1351,7 +1351,7 @@ export const getMaxGasLimit = (blockGasLimit: bigint): bigint => {
   // million will have a max batch size of 5.68 million (= 80% * 7.1 million), which is too low.
   if (blockGasLimit <= BigInt(8_500_000)) {
     return blockGasLimit
-  } else if (blockGasLimit <= BigInt(13_500_000)) {
+  } else if (blockGasLimit <= BigInt(20_000_000)) {
```

Comment:
Can I get some context on this? Or is this temporary?

Reply:
This is temporary while we work out a more comprehensive solution.